examples/customized_models/HGT/hgt_nc.py

```diff
@@ -368,7 +368,8 @@ def main(args):
     # Run inference on the inference dataset and save the GNN embeddings in the specified path.
     infer.infer(dataloader, save_embed_path=config.save_embed_path,
                 save_prediction_path=config.save_prediction_path,
-                use_mini_batch_infer=True)
+                use_mini_batch_infer=True,
+                save_embed_format=config.save_embed_format)
```

Comment:
you don't have to change here. the argument should have a default value that saves the embeddings with pytorch format.